examples/MockRequest.hs

```diff
@@ -3,7 +3,10 @@
 module Main where
 
 import           PushyClient                     (makePushyPostRequest)
-import           PushyClient.Types.PushyRequest  (defaultPushyPostRequestBody)
+import           PushyClient.Types.PushyRequest  (IosNotification (..),
```

Comment:
I'm not sure if I like the cases here `IosNotification` looks nothing I've ever seen. I would consider using `IOSNotification`.

Reply:
I just realized that that would be breaking change for public api.

Reply:
@turboMaCk true, but as far as I know, we're the only ones using this package right now. Generally, you'd be notified if someone cloned the repository to use in their project?

Reply:
nope. You'll never know and you should not know. License permits code to be used by anyone as long as the license file itself is included. If using the library would track user you would need some terms of usage where user agrees to be tracked. The only thing you can look at is number of downloads from hackage but those won't say much. Many of these can be stackage pulling code from hackage etc.

Reply:
Ahh got it. Currently the library hasn't been uploaded to stackage though, so one would have to use the library in their project by referencing a specific commit. So I suppose the changes shouldn't affect their project as much?

Reply:
I think in that case it's fine to break the public api.